repos/README.md

```diff
@@ -0,0 +1,8 @@
+# Repo CRD
+
+A custom resource definition for Kubernetes.
+
+- it allows the user to define a resource where they can store application sources and/or artifacts
+- once the resource is created, the user is presented with a protocol and location for uploading (in status)
```

Comment:
```
- once the resource is created, the user is presented with a protocol and location for uploading (in status)
```
Should we return an url to upload data, or just give ways to define where the sources are coming from?
Ideally, as an input I should be able to pick a supported `Repository` getter (that those can be extended to provide support to different source inputs, e.g. Github)

Reply:
I think we need to assume there's nothing else required outside of this.
If we need another source to pull from, we're shifting responsibility.
So given the user has sources, we provide a place to put them.